### PR TITLE
fixes 36

### PR DESCRIPTION
**Pull request by Sparrow0hawk**
added page in on using temporary storage on compute nodes

- updated toc
- updated usage start page

### CODE REVIEW

**Comment by Sparrow0hawk** [comment]
Also now updated the link in the gaussian page

File: book/usage/scratch.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+* - Option
+  - Description
+  - Default
+* - `-l disk=<bytes>`
+  - Sets the limit of the local disk available under `$TMPDIR` (per core)
+  - 1G per core
+* - `-l disk_type=*|ssd|hdd`
+  - Specifies the type of local disk that files under `$TMPDIR` will be written to. There are 3 options:
+
+    1. which is the default and specifies any type of disk
```
Comment (jhodrien) [inline book/usage/scratch.md:100]:
```suggestion
    1. \* which is the default and specifies any type of disk
```

File: book/usage/scratch.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    2. ssd – which explicitly specifies solid state disk
+    3. hdd – which explicitly specifies hard disk drive.
+
+    Reading a writing will be much quicker to a solid state drive and this option should be used if you have lots of read and write operations.
```
Comment (jhodrien) [inline book/usage/scratch.md:104]:
```suggestion
    Reading and writing will be much quicker to a solid state drive and this option should be used if you have lots of read and write operations.
```

File: book/usage/scratch.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+  - 1
```
Comment (jhodrien) [inline book/usage/scratch.md:105]:
```suggestion
  - \*
```

**Review by jhodrien — CHANGES_REQUESTED** [review]
Otherwise approve

**Pull request merged**